SharedCode/src/commonMain/kotlin/http/RequestLock.kt

```diff
@@ -0,0 +1,33 @@
+package com.jetbrains.handson.mpp.mobile.http
+
+import com.soywiz.klock.DateTime
+import kotlin.random.Random
+
+class RequestLock {
```

Comment:
I think this class could benefit from some comments describing its intent. Without the context of our earlier discussion it's not immediately obvious what the reason for it is.